code-challenges.js

```diff
@@ -32,6 +61,27 @@ var hand4 = [3, 5, 3, 5, 3]
 // Expected output: true
 
 
+// Create a function that takes in an array as an argument
+const fullHouse = (arr) => {
+    // Sort the array in order
+    let orderArr = arr.sort((a,b)=>a-b)
+    // Check to see if the values are equal to one another (0-2 && 3-4 || 0-1 && 2-4)
+    if (orderArr[0] === orderArr[1] && orderArr[0] === orderArr[2] && orderArr[3] === orderArr[4]) {
```

Comment:
Also consider using a data structure that can model a full house or provide a comment describe why these conditions are necessary.